python-package/securexgboost/core.py

```diff
@@ -2339,143 +2629,245 @@ def get_split_value_histogram(self, feature, fmap='', bins=None, as_pandas=True)
 # APIs invoked by RPC server
 ##########################################
 
+def py2c_sigs(signatures, sig_lengths):
```

Comment:
Let's move this function to the top of `core.py` where all the other utils functions are